### PR TITLE
feat(button): add textAlign attribute

### PR DESCRIPTION
**Pull request by skhamvon**
ODS-870
Add textAlign to button to set the position of the text of the unnamed slot

### CODE REVIEW

File: packages/components/button/src/components/osds-button/interfaces/attributes.ts [hunk 1]
```diff
@@ -26,6 +27,8 @@ interface OdsButtonAttribute {
   /** link target if we are in a link-mode (using `href` attribute) */
   target?: OdsHTMLAnchorElementTarget;
   /** type of the vanilla button */
```
Comment (aesteves60) [inline packages/components/button/src/components/osds-button/interfaces/attributes.ts:29]: Comment need to be above type

File: packages/components/button/src/components/osds-button/interfaces/attributes.ts [hunk 2]
```diff
@@ -26,6 +27,8 @@ interface OdsButtonAttribute {
...
+
```
Comment (aesteves60) [inline packages/components/button/src/components/osds-button/interfaces/attributes.ts:30]: miss comment

File: packages/components/button/src/components/osds-button/constants/button-textalign.ts [hunk 3]
```diff
@@ -0,0 +1,12 @@
+enum ODS_BUTTON_TEXTALIGN {
```
Comment (aesteves60) [inline packages/components/button/src/components/osds-button/constants/button-textalign.ts:1]: I think it's should be `ODS_BUTTON_TEXT_ALIGN`. @dpellier what do you think ?

**Review by aesteves60 — CHANGES_REQUESTED** [review]

File: packages/components/button/src/components/osds-button/interfaces/attributes.ts (hunk 2 above)
Comment (dpellier) [inline packages/components/button/src/components/osds-button/interfaces/attributes.ts:30]: remove blank line

**Comment by ovh-cds** [comment]
CDS Report build-storybook#1381.0 ✘  
* 
  * build ✘

File: packages/components/button/src/components/osds-button/interfaces/attributes.ts (hunk 2 above)
Comment (skhamvon) [reply packages/components/button/src/components/osds-button/interfaces/attributes.ts]: Removed

**Review by dpellier — APPROVED** [review]

**Review by astagnol — APPROVED** [review]

**Pull request merged**